backend/src/nodes/properties/inputs/image_dropdown_inputs.py

```diff
@@ -93,7 +93,31 @@ def InterpolationInput() -> DropDownInput:
                 "option": "Cubic",
                 "value": InterpolationMethod.CUBIC,
             },
-            {"option": "Lanczos", "value": InterpolationMethod.LANCZOS},
+            {
+                "option": "Lanczos",
+                "value": InterpolationMethod.LANCZOS,
+            },
+        ],
+    )
+
+
+def ReducedInterpolationInput() -> DropDownInput:
```

Comment:
Reduce?

Reply:
Reduced, not Reduce, because PIL rotate accepts a reduced set of the interpolation methods. I renamed it to RotateInterpolationInput.